psi4/driver/procrouting/proc_util.py

```diff
@@ -221,4 +221,4 @@ def print_ci_results(ciwfn, rname, scf_e, ci_e, print_opdm_no=False):
         ciwfn.print_vector(dvec, root)
 
     # True to keep the file
-    #dvec.close_io_files(True)
+    dvec.close_io_files(True)
```

Comment:
Well it *is* something screwy in PSIO. For some reason that line wipes deletes that file in certain conditions causing SAPT(CASSCF) to fail. Looks like I accidentally committed that at one point, causing others to fail. This *should* be the right way to do it however.